SPECS/libconfig/libconfig.spec

```diff
@@ -56,6 +56,9 @@ cd ..
 %{_libdir}/pkgconfig/libconfig*.pc
 
 %changelog
+* Tue Jul 06 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 1.7.3-2
+- Removing minor version number from the %%files section.
+
```

Comment:
cosmetic nitpick :)
```suggestion
```

Reply:
Not sure what you mean. All changelog entries are separated by an empty line in this spec and I believe that's the way we'd prefer.

Reply:
apologies .. was using github cli .. and this did not show as intended.